extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -79,13 +90,19 @@ export class RemoteQueriesManager {
       const shouldOpenView = await showInformationMessageWithAction(message, 'View');
       if (shouldOpenView) {
         await this.interfaceManager.showResults(query, queryResult);
-
       }
-    } else if (queryResult.status === 'CompletedUnsuccessfully') {
-      await showAndLogErrorMessage(`Remote query execution failed. Error: ${queryResult.error}`);
-      return;
-    } else if (queryResult.status === 'Cancelled') {
+    } else if (queryWorkflowResult.status === 'CompletedUnsuccessfully') {
+      await showAndLogErrorMessage(`Remote query execution failed. Error: ${queryWorkflowResult.error}`);
+
+    } else if (queryWorkflowResult.status === 'Cancelled') {
       await showAndLogErrorMessage('Remote query monitoring was cancelled');
+
+    } else if (queryWorkflowResult.status === 'InProgress') {
```

Comment:
Since this should never happen, why is it not in the `else` clause (L105)?

Reply:
In order for `assertNever` to work, all cases need to be explicitly handled. Otherwise, `assertNever` will have a compile error.